import-export-cli/cmd/exportApps.go

```diff
@@ -0,0 +1,103 @@
+)
+
+var exportAppsWithKeys bool
+var exportAppsFormat string
+var startFromBeginningForApps bool
+var isProcessCompletedForApps bool
+
+// ExportApps command related usage info
+const ExportAppsCmdLiteral = "apps"
+const exportAppsCmdShortDesc = "Export Apps for migration"
+
+const exportAppsCmdLongDesc = "Export Applications of a given tenant from a specified  environment"
+
+const exportAppsCmdExamples = utils.ProjectName + ` ` + ExportCmdLiteral + ` ` + ExportAppsCmdLiteral + ` -e dev --force
+` + utils.ProjectName + ` ` + ExportCmdLiteral + ` ` + ExportAppsCmdLiteral + ` -e prod
+NOTE: The flag (--environment (-e)) is mandatory`
+
+// ExportAppsCmd represents the exportApps command
+var ExportAppsCmd = &cobra.Command{
+	Use: ExportAppsCmdLiteral + " (--environment <environment-from-which-the-app-should-be-exported> " +
+	"--format <export-format> --force)",
+	Short:   exportAppsCmdShortDesc,
+	Long:    exportAppsCmdLongDesc,
+	Example: exportAppsCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + ExportAppsCmdLiteral + " called")
+		var appsExportDirectoryPath = filepath.Join(utils.ExportDirectory, utils.ExportedMigrationArtifactsDirName)
+
+		cred, err := GetCredentials(CmdExportEnvironment)
+		if err != nil {
+			utils.HandleErrorAndExit("Error getting credentials", err)
+		}
+		executeExportAppsCmd(cred, appsExportDirectoryPath)
+	},
+}
+
+func executeExportAppsCmd(credential credentials.Credential, appsExportDirectoryPath string) {
+    // Create dir structure
+    appExportDir := impl.CreateExportAppsDirStructure(appsExportDirectoryPath, CmdResourceTenantDomain, CmdExportEnvironment,
+        CmdForceStartFromBegin)
+    exportRelatedFilesPath := filepath.Join(appsExportDirectoryPath, CmdExportEnvironment,
+        utils.GetMigrationExportTenantDirName(CmdResourceTenantDomain))
+    // e.g. /home/user/.wso2apictl/exported/migration/production/wso2-dot-org
+    startFromBeginningForApps = false
+    isProcessCompletedForApps = false
+
+    fmt.Println("\nExporting Apps for the migration...")
+    if CmdForceStartFromBegin {
+        startFromBeginningForApps = true
+    }
+
+    if (utils.IsFileExist(filepath.Join(exportRelatedFilesPath, utils.LastSucceededAppFileName))) && !startFromBeginningForApps {
+        impl.PrepareResumptionForApps(credential, exportRelatedFilesPath, CmdResourceTenantDomain, CmdUsername, CmdExportEnvironment)
+    } else {
+        impl.PrepareStartAppsFromBeginning(credential, exportRelatedFilesPath, CmdResourceTenantDomain, CmdUsername, CmdExportEnvironment)
+    }
+
+    impl.ExportApps(credential, exportRelatedFilesPath, CmdExportEnvironment, CmdResourceTenantDomain, exportAppsFormat,
+     CmdUsername, appExportDir, exportAppsWithKeys)
+}
+
+// Init using Cobra
+func init() {
+	ExportCmd.AddCommand(ExportAppsCmd)
+    ExportAppsCmd.PersistentFlags().BoolVarP(&CmdForceStartFromBegin, "force", "", false,
```

Comment:
Shall we fix indentation here?

```suggestion
	ExportAppsCmd.PersistentFlags().BoolVarP(&CmdForceStartFromBegin, "force", "", false,
```